quinn-h3/examples/h3_server.rs

```diff
@@ -62,12 +63,13 @@ async fn main() -> Result<()> {
         println!("server received connection");
         let connection = connecting
             .await
-            .map_err(|e| anyhow!("accept failed: {:?}", e))
-            .expect("server failed");
+            .map_err(|e| error!("accept failed: {:?}", e));
```

Comment:
Why do we no longer fail if the connection establishment fails? Perhaps we should at least log/print something in this case?

Reply:
We want to continue accepting the following incoming connections, so we don't want to fail `main()`. 

I guess there is something logged with the `error!()` call. 

Writing this with `if let` is a bit convoluted. I'll make this a `match`, will read much better.

Reply:
Ah, if `error!()` logs then we should be fine.

Reply:
Rewrote it using `match`